src/apps/vpn/ui/screens/settings/ViewNotifications.qml

```diff
@@ -53,12 +33,10 @@ VPNViewBase {
             //% "Get notified if a guest Wi-Fi portal is blocked due to VPN connection"
             subLabelText: qsTrId("vpn.settings.guestWifiAlert.description")
             isChecked: (VPNSettings.captivePortalAlert)
-            enabled: vpnFlickable.vpnIsOff
+            enabled: true
```

Comment:
We can just remove this line and the other instances of `enabled: true` since these are enabled by default.